docs/app/Components/Sidebar/Sidebar.js

```diff
@@ -51,12 +51,21 @@ export default class Sidebar extends Component {
 
   componentDidMount() {
     document.addEventListener('keydown', this.handleDocumentKeyDown)
+    this.setSearchInput()
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    this.setSearchInput()
   }
 
   componentWillUnmount() {
     document.removeEventListener('keydown', this.handleDocumentKeyDown)
   }
 
+  setSearchInput() {
+    this._searchInput = findDOMNode(this).querySelector('.ui.input input')
```

Comment:
Input is now a stateless component, so no `refs` are available.  Updated the doc sidebar to use a regular query selector instead.